DependencyInjection/Configuration.php

```diff
@@ -103,6 +103,7 @@ public function getConfigTreeBuilder()
                     ->addDefaultsIfNotSet()
                     ->children()
                         ->scalarNode('default_format')->defaultNull()->end()
+                        ->scalarNode('method_prefix')->defaultTrue()->end()
```

Comment:
By just looking at the name of the option I would expect to be able to pass a string that will be the prefix. To have an option acting as a toggle to turn on/off the feature I would rather rename it to something like `prefix_methods`.